src/MatchedLearnerApi/Startup.cs

```diff
@@ -57,11 +97,25 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
 
             app.UseRouting();
 
+            app.UseAuthentication();
             app.UseAuthorization();
 
             app.UseEndpoints(endpoints =>
             {
                 endpoints.MapControllers();
+                endpoints.MapHealthChecks("/health", new HealthCheckOptions
+                {
+                    ResponseWriter = HealthCheckResponseWriter.WriteJsonResponse,
+                });
+                endpoints.MapHealthChecks("/ping", new HealthCheckOptions
+                {
+                    Predicate = (_) => false,
+                    ResponseWriter = (context, report) =>
+                    {
+                        context.Response.ContentType = "application/json";
+                        return context.Response.WriteAsync("");
+                    }
+                });
```

Comment:
Assume these are the standard das healthchecks? devops will have an opinion on this and they may need to change if this didn't come from devops

Reply:
this is standard DAS taken from one of the other API's